CONTRIBUTING.md

```diff
@@ -65,3 +54,55 @@ For new features that affect test coverage, use a `coverage/*` branch in this re
 ## Writing a new test
 
 See the [Integration Tests CONTRIBUTING.md](./packages/openactive-integration-tests/CONTRIBUTING.md) for guidance on writing a new test.
+
+## Locally running Test Suite to test changes
+
+When you are making changes to the Test Suite, please run it on your machine to check that the changes work before submitting a [pull request](#pull-request-process). These changes can be checked against the [**reference implementation**](#reference-implementation). You may or may not also have changes to the reference implementation that is required for your updated tests to pass. Either way, you'll need to run both projects on your machine order to test your changes.
+
+How to run them both locally:
+
+* **Reference Implementation**: To run this locally, follow the guidelines in its [project's contribution documentation](https://github.com/openactive/OpenActive.Server.NET/blob/master/CONTRIBUTING.md).
```

Comment:
Included in this PR: https://github.com/openactive/OpenActive.Server.NET/pull/216